webapp/home/templates/home/snippets/footer.html

```diff
@@ -11,50 +11,14 @@
 
   <div class="container logo">
     <div class="text-center">
-      <a href="https://www.biocommons.org.au" target="_blank">
-        <img src="{% static 'logos/footer/au-biocommons-2.png' %}" />
+      <a href="https://www.education.gov.au/ncris" target="_blank">
+        <img src="{% static 'logos/footer/NCRIS-PROVIDER.png' %}" />
       </a>
-
-      <a href="https://www.qcif.edu.au" target="_blank">
-        <img src="{% static 'logos/footer/qcif-2.png' %}" />
-      </a>
-
-      <a href="https://www.unimelb.edu.au" target="_blank">
-        <img src="{% static 'logos/footer/unimelb-2.svg' %}" />
-      </a>
-
-      <a href="https://www.aarnet.edu.au" target="_blank">
-        <img src="{% static 'logos/footer/aarnet.png' %}" />
-      </a>
-    </div>
-
-    <div class="text-center second">
-      <a href="https://www.uq.edu.au" target="_blank">
-        <img src="{% static 'logos/footer/uq-grayscale.png' %}" />
-      </a>
-
-      <a href="https://ardc.edu.au" target="_blank">
-        <img src="{% static 'logos/footer/ardc-grayscale.png' %}" />
+      <a href="https://bioplatforms.com/" target="_blank">
+        <img src="{% static 'BIO-RGB_Full-POS.png' %}" />
       </a>
-
-      <a href="https://bioplatforms.com" target="_blank">
-        <img src="{% static 'logos/footer/bpa-grayscale.png' %}" />
-      </a>
-
-      <a href="https://nci.org.au" target="_blank">
-        <img src="{% static 'logos/footer/nci-grayscale.png' %}" />
-      </a>
-
-      <a href="https://pawsey.org.au" target="_blank">
-        <img src="{% static 'logos/footer/pawsey-grayscale.png' %}" />
-      </a>
-
-      <a href="https://www.dese.gov.au/ncris" target="_blank">
-        <img src="{% static 'logos/footer/ncris-grayscale.png' %}" />
-      </a>
-
-      <a href="https://www.qld.gov.au" target="_blank">
-        <img src="{% static 'logos/footer/qld-gov.png' %}" />
+      <a href="https://www.biocommons.org.au/" target="_blank">
+        <img src="{% static 'Australian-Biocommons-Logo-Tagline-Infrastructure-RGB-no-padding.png' %}" />
```

Comment:
~Image path is missing `/logos/footer/`~ Fixed

Reply:
Does this extra text really need to be squeezed into the Biocommons logo? If it's important, can we put this text somewhere else in the footer where it is more legible, and doesn't look like a fourth logo?

![image](https://github.com/user-attachments/assets/d9ff0961-886a-4d14-b0e7-21a447465ab6)

Reply:
Hmm I will raise this one, you're right it is rather small and hard to read in this position

Reply:
Ok I've reverted to the regular BioCommons logo and moved the text into the footer. I removed the header of 'acknowledgements' as they're really 3 distinct statements, and the header takes up space that we need to separate the statements.

@neoformit do the <br> tags that I put leave a white line between the sentence? Or do I need to put a <p></p> in to force a white line between each line of text?

Reply:
Ha, I see my comment actually formatted the code and answered my own question! Updated the footer code so I think it's good to go now :)

Reply:
Haha, yes! `<br>` is a line break, `<p>` is a paragraph. Usually a blank line between text blocks means that it's a new paragraph, so if that's the case then use `<p>` instead.

Reply:
This is ok:

![image](https://github.com/user-attachments/assets/66301286-f361-4fa5-a69c-8d5918fe9213)

But I have shuffled around the columns and spacing and I think this looks a bit neater? What do you think? Does the "Acknowledgements" title still make sense? It just looks out-of-alignment if one section doesn't have a title.

![image](https://github.com/user-attachments/assets/9a9a41ce-7ea0-4f38-9113-0b35596c002f)

Reply:
Agreed it does look a bit odd with one section without a title, so let's leave the title there. I like your spacing shuffle, thanks for that. I'll put in more changes to fix the paragraph code, and reinstate the title, can you then please make the alignment fix?

Are the words 'partner organisations' in the second acknowledgement hyperlinked to /about#operational-partners? That's what I've tried to achieve but it's not coloured like a hyperlink - maybe that's intentional in the footer?

Reply:
Cool, I've merged our changes. I think it looks decent now:

![image](https://github.com/user-attachments/assets/aeb92d59-56a3-41eb-a618-3f84f34a420a)

Reply:
Agreed - I'll email Gareth and Christina to confirm they're happy with it